tools/vsextension/ItemTemplates/oehost/host.c

```diff
@@ -0,0 +1,89 @@
+#include <openenclave/host.h>
+#include "$enclavename$_u.h"
+
+oe_result_t create_$enclavename$_enclave(const char* enclave_name, oe_enclave_t** out_enclave)
+{
+    oe_enclave_t* enclave = NULL;
+    uint32_t enclave_flags = 0;
+    oe_result_t result;
+
+    *out_enclave = NULL;
+
+    // Create the enclave
+#ifdef _DEBUG
+    enclave_flags |= OE_ENCLAVE_FLAG_DEBUG;
+#endif
+    result = oe_create_$enclavename$_enclave(
+        enclave_name,
+        OE_ENCLAVE_TYPE_AUTO,
+        enclave_flags,
+        NULL,
+        0,
+        &enclave);
+    if (result != OE_OK)
+    {
+        printf("Error %d creating enclave, trying simulation mode...\n", result);
+        enclave_flags |= OE_ENCLAVE_FLAG_SIMULATE;
+        result = oe_create_$enclavename$_enclave(
+            enclave_name,
+            OE_ENCLAVE_TYPE_AUTO,
+            enclave_flags,
+            NULL,
+            0,
+            &enclave);
+    }
+    if (result != OE_OK)
+    {
+        return result;
+    }
+
+    *out_enclave = enclave;
+    return OE_OK;
+}
+
+void sample_enclave_call(void)
+{
+    oe_enclave_t* enclave = NULL;
+    oe_result_t result = create_$enclavename$_enclave(
+#ifdef OE_USE_OPTEE
+        "$enclaveguid$",
+#else
+        "$enclavename$.elf.signed",
```

Comment:
The samples published with OE for Linux generate enclave binaries without the `.elf` part.

The unsigned enclave is just named `$enclavename$, and the signed enclave is named `$enclavename$.signed`.

I suppose we'd want to generate binaries with the same naming template within the extension?